### ISSUE TITLE
Typedefs like Lua plugins

### ISSUE THREAD

**Issue by Minh9897** [Kong/kong-python-pdk#73]
Does this python pdk support typedef to validate Url like in Lua Plugins ?

**Comment by fffonion** [comment]
Hi @Minh9897 this is currently not supported, you can use python libraries to the validation on python land.

**Comment by Minh9897** [comment]
If I validate in class Plugin, the plugin still create even the value is invalid. Am i right?

**Comment by fffonion** [comment]
Yes, currently we don't support validation on creation/update time in external plugins.
